### ISSUE TITLE
using shift-select as first action crashes editor 

### ISSUE THREAD

**Issue by saltyElefant** [ddnet/ddnet#5640]
Restart client, open any map and do a random shift-select. This will crash the 16.2.2 client everytime.
If you select a tile with a different id beforehand it won't crash.

Doesn't crash with 16.1.

**Comment by def-** [comment]
Also reported by bencie on Discord, I can reproduce it:
```
Thread 1 "DDNet" received signal SIGSEGV, Segmentation fault.
0x0000555555759b5f in CEditor::DoMapEditor (this=0x55555592dfe0, View=...) at /media/ddnet/src/game/editor/editor.cpp:2639
2639								apEditLayers[k]->FillSelection(m_Brush.IsEmpty(), m_Brush.m_vpLayers[BrushIndex], r);
(gdb) bt
#0  0x0000555555759b5f in CEditor::DoMapEditor(CUIRect) (this=0x55555592dfe0, View=...) at /media/ddnet/src/game/editor/editor.cpp:2639
#1  0x000055555575abb4 in CEditor::Render() (this=this@entry=0x55555592dfe0) at /media/ddnet/src/game/editor/editor.cpp:5745
#2  0x000055555575b334 in CEditor::OnRender() (this=0x55555592dfe0) at /media/ddnet/src/game/editor/editor.cpp:6435
#3  0x00005555556421f9 in CClient::Run() (this=this@entry=0x7fffe4ad8010) at /media/ddnet/src/engine/client/client.cpp:3374
#4  0x0000555555582667 in main(int, char const**) (argc=<optimized out>, argv=<optimized out>) at /media/ddnet/src/engine/client/client.cpp:4762
```
> On latest nightly, steam version 16.2.1-20220726, when you shift hold nothing in editor (to delete tiles) game crashes

**Comment by def-** [comment]
Duplicate of #5626, will be fixed by https://github.com/ddnet/ddnet/pull/5627

**Comment by Robyt3** [comment]
Fixed by #5627
